src/sass/layout/_header.scss

```
@@ -0,0 +1,168 @@
+// Logo header
+.header-logo img,
+.logo-modal img {
+  @media (min-width: $tablet-width) {
```

В папке `sass/utils/_breakpoints.sass` прописаны правила для брекпоинтов

```suggestion
  @include for-size(tablet) {
```

---

по всем твоим sass файлам необходимо просмотреть и исправить